llama_index/chat_engine/simple.py

```diff
@@ -1,117 +1,69 @@
-from typing import Any, Optional
-
-from llama_index.bridge.langchain import BaseChatModel, ChatGeneration
+from typing import Any, List, Optional
 
 from llama_index.chat_engine.types import BaseChatEngine, ChatHistoryType
-from llama_index.chat_engine.utils import (
-    is_chat_model,
-    to_chat_buffer,
-    to_langchain_chat_history,
-)
 from llama_index.indices.service_context import ServiceContext
 from llama_index.llm_predictor.base import LLMPredictor
-from llama_index.prompts.base import Prompt
-from llama_index.prompts.prompt_type import PromptType
+from llama_index.llms.base import LLM, ChatMessage
 from llama_index.response.schema import RESPONSE_TYPE, Response
 
-DEFAULT_TMPL = """\
```

Comment:
Wasn't it useful to have a way to control the overall behavior of the chat?

Reply:
ya left it as a TODO below. Thinking about configuring it at the LLM level.